recipes/highfive/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "HighFive" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "deee33d7f578e33dccb5d04771f4e01b89a980dd9a3ff449dd79156901ee8d25" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: HighFive-v2.0.tar.gz
+  url: https://github.com/BlueBrain/HighFive/archive/v2.0.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win and py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - hdf5
+    - boost
```

Comment:
Are you sure you need `boost`? If so, then `boost-cpp` can be removed afaik.

Reply:
it seems so, removed boost-cpp